sdk/tables/Azure.Data.Tables/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Thank you to our developer community members who helped to make Azure Tables bet
 ### Changed
 
 - Removed the `Timestamp` property from the serialized entity when sending it to the service as it is ignored by the service (A community contribution, courtesy of _[joelverhagen](https://github.com/joelverhagen))_
+- Fixed handling of paging headers when Table Storage returned a `x-ms-continuation-NextPartitionKey` but no `x-ms-continuation-NextRowKey`. This was causing an HTTP 400 Bad Request on the subsequent page query.
```

Comment:
Sorry I don't know the patterns here but I added this to try to fix a `dotnet pack` error.

Reply:
Thanks - this should just be moved under a new header section called `### Key Bug Fixes`

Reply:
oh, and you may as well add your attribution tag like I did for your last PR 💯 😃